assets/i18n/strings.i18n.json

```
@@ -123,7 +123,7 @@
 
         "doneButton": "Done",
 
-        "default": "Default",
+        "defaultText": "defaultText",
```

defaultText... *is this intended?*

```suggestion
        "defaultText": "Default",
```